config/application.rb

```diff
@@ -8,7 +8,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-Dotenv::Railtie.load
```

Comment:
This is to fix a deprecation message: DEPRECATION WARNING: Dotenv::Railtie is deprecated! Use Dotenv::Rails instead. (called from <top (required)> at /Users/crowesn/Development/treatment_database/config/application.rb:11)